herd/AArch64Sem.ml

```diff
@@ -1092,7 +1103,16 @@ module Make
                 fun ma _a ->
                 mop Access.PHY ma
                 >>= M.ignore >>= B.next1T in
-            lift_kvm dir updatedb mop ma an ii mphy
+            if self then
```

Comment:
Do you need to check for self? I am thinking that if there is a memory instruction to a location then we better be using self, otherwise we shouldn't get to this point at all.